test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/mq_as_scope_in_v1_queries.test

```diff
@@ -183,7 +183,7 @@ Date: REGEX(.*)
 05. GET entities matching q=A1==1 AND mq=A1.M1==2, see E2, using /v1/queryContext
 =================================================================================
 HTTP/1.1 200 OK
-Content-Length: 246
+Content-Length: 245
```

Comment:
This is due to a "\n" has been removed in NGSIv1 rendering. That "\n" was probably forgotten when the de-indent work in NGSIv1 was done.

Reply:
NTC